package.json

```diff
@@ -74,15 +74,21 @@
 		"update:wpcli": "bin/wp-cli.phar cli update --nightly --yes && bin/wp-cli.phar --info",
 		"prewp": "npm run install:wpcli && npm run update:wpcli",
 		"wp": "php -d memory_limit=4G bin/wp-cli.phar",
-		"makepot:android": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.android.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./gutenberg.pot --ignore-domain gutenberg-android.pot",
-		"makepot:ios": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./gutenberg.pot --ignore-domain gutenberg-ios.pot",
+		"makepot:android:gutenberg": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.android.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./gutenberg.pot --ignore-domain gutenberg-android.pot",
+		"makepot:android:jetpack": "npm run wp -- i18n make-pot ./jetpack/projects/plugins/jetpack/extensions --include=*.native.js,*.android.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./jetpack.pot --ignore-domain jetpack-android.pot",
+		"makepot:android:layout-grid": "npm run wp -- i18n make-pot ./block-experiments/blocks/layout-grid --include=*.native.js,*.android.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./layout-grid.pot --ignore-domain layout-grid-android.pot",
+		"makepot:ios:gutenberg": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./gutenberg.pot --ignore-domain gutenberg-ios.pot",
+		"makepot:ios:jetpack": "npm run wp -- i18n make-pot ./jetpack/projects/plugins/jetpack/extensions --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./jetpack.pot --ignore-domain jetpack-ios.pot",
+		"makepot:ios:layout-grid": "npm run wp -- i18n make-pot ./block-experiments/blocks/layout-grid --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./layout-grid.pot --ignore-domain layout-grid-ios.pot",
```

Comment:
Generate POT files for each plugin including Gutenberg. This approach won't scale easily as we add more plugins, so I'll provide in a new PR a way to unify the POT files creation in a single command.